saltyrtc/server/__init__.py

```diff
@@ -4,6 +4,16 @@
 """
 import itertools
 
+from . import (
```

Comment:
This relanded apparently (?)

Reply:
Ah, the isort commit. Good catch.

Reply:
Ok, now the mypy error is back:

```python
[maintenance-2021]danilo@c3po:server-python$ MYPYPATH=${PWD}/stubs mypy saltyrtc examples
saltyrtc/server/protocol.py:19: error: Cannot find implementation or library stub for module named 'websockets'
saltyrtc/server/protocol.py:19: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
saltyrtc/server/server.py:24: error: Cannot find implementation or library stub for module named 'websockets'
saltyrtc/server/server.py:26: error: Cannot find implementation or library stub for module named 'websockets.typing'
saltyrtc/server/__init__.py: error: Source file found twice under different module names: 'server' and 'saltyrtc.server'
Found 4 errors in 3 files (errors prevented further checking)
```

Since it seems to work fine _with_ the imorts, should we I keep them? (I'd move the import change to the proper commit though.)

Reply:
Ok